crmsh/ocfs2.py

```diff
@@ -0,0 +1,289 @@
+import re
+from contextlib import contextmanager
+from . import utils
+from . import bootstrap
+from . import ra
+
+
+class OCFS2Manager(object):
+    """
+    Class to manage OCFS2 and related resources
+    """
+    RA_ID_PREFIX = "ocfs2-"
+    DLM_RA_ID = "{}dlm".format(RA_ID_PREFIX)
+    FS_RA_ID = "{}clusterfs".format(RA_ID_PREFIX)
+    LVMLOCKD_RA_ID = "{}lvmlockd".format(RA_ID_PREFIX)
+    LVMACTIVATE_RA_ID = "{}lvmactivate".format(RA_ID_PREFIX)
+    GROUP_ID = "{}group".format(RA_ID_PREFIX)
+    CLONE_ID = "{}clone".format(RA_ID_PREFIX)
+    VG_ID = "{}vg".format(RA_ID_PREFIX)
+    LV_ID = "{}lv".format(RA_ID_PREFIX)
+
+    MAX_CLONE_NUM = 8
+    MKFS_CMD = "mkfs.ocfs2 --cluster-stack pcmk --cluster-name {} -N {} -x {}"
+    HINTS_WHEN_RUNNING = """
+Cluster service already successfully started on this node except configuring OCFS2
```

Comment:
I don't get it's meaning. Can you explain a little ？

Reply:
This will be shown when calling `crm cluster init` with -o option, when meeting errors while configuring ocfs2, on this time, cluster service was already up, so I want to give user a hint that no need to setup the whole cluster from beginning next time,
just using ocfs2 stage to configure ocfs2 separately

Reply:
I see. How about to rephrase it as something like,
"Cluster service already successfully initialized on this node except, but lack of prerequisites to continue configuring OCFS2. Please fix it and use the stage procedure to configure ocfs2 seperately, eg. `crm cluster init ocfs2 xxx`"

Reply:
Changed as
```
The cluster service has already been initialized, but the prerequisites are missing
to configure OCFS2. Please fix it and use the stage procedure to configure OCFS2 separately,
e.g. crm cluster init ocfs2 -o <ocfs2_device>
```